dynamiqs/solvers/solver.py

```diff
@@ -38,6 +36,11 @@ def __init__(
         self.exp_ops = exp_ops
         self.options = options
 
+        # aliases
+        self.cdtype = self.options.cdtype
+        self.rdtype = self.options.rdtype
+        self.device = self.options.device
+
```

Comment:
Not sure about this. Don't you think it's risky, in case one is modified and not the other?

Reply:
We could do better by having only a property, but honestly it's internal I think it's fine.